core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -210,25 +211,31 @@ private void configureClientAuth(SslContextFactory sslContextFactory, RestConfig
                 RestConfig.SSL_CLIENT_AUTH_CONFIG,
                 RestConfig.SSL_CLIENT_AUTHENTICATION_CONFIG
         );
-        clientAuthentication = config.getBoolean(RestConfig.SSL_CLIENT_AUTH_CONFIG)
-                ? RestConfig.SSL_CLIENT_AUTHENTICATION_REQUIRED
-                : RestConfig.SSL_CLIENT_AUTHENTICATION_NONE;
+        clientAuthentication = config.getString(RestConfig.SSL_CLIENT_AUTH_CONFIG);
       }
     }
 
-    switch (clientAuthentication) {
+    switch (clientAuthentication.toLowerCase(Locale.ROOT)) {
```

Comment:
Given all acceptable values are in the ASCII range, do we really need to use locale?

Reply:
Nevermind. This very correct. My mistake. :?